src/program/packetblaster/replay/replay.lua

```diff
@@ -0,0 +1,88 @@
+-- Use of this source code is governed by the Apache 2.0 license; see COPYING.
+
+module(..., package.seeall)
+
+local engine    = require("core.app")
+local config    = require("core.config")
+local timer     = require("core.timer")
+local pci       = require("lib.hardware.pci")
+local intel10g  = require("apps.intel.intel10g")
+local intel_app = require("apps.intel.intel_app")
+local basic_apps = require("apps.basic.basic_apps")
+local main      = require("core.main")
+local PcapReader= require("apps.pcap.pcap").PcapReader
+local LoadGen   = require("apps.intel.loadgen").LoadGen
+local lib = require("core.lib")
+local ffi = require("ffi")
+local C = ffi.C
+
+local usage = require("program.packetblaster.replay.README_inc")
+
+local long_opts = {
+   duration     = "D",
+   help         = "h",
+   src          = "s",
+   dst          = "d",
+   sizes        = "S"
+}
+
+function run (args)
+   local opt = {}
+   local mode = table.remove(args, 1)
+   local duration
+   local c = config.new()
+   function opt.D (arg) 
+      duration = assert(tonumber(arg), "duration is not a number!")  
+   end
+   function opt.h (arg)
+      print(usage)
+      main.exit(1)
+   end
+
+   args = lib.dogetopt(args, opt, "hD:", long_opts)
```

Comment:
`long_opts` defines more options (`src`, `dst`, `sizes`) than short options "hD:" (help, duration). It seems this was already present in master, but since you're moving the code I think is nice to fix it :)